css/style.css

```diff
@@ -6,3 +6,49 @@
  *    for example: General styles, Navigation styles, Hero styles, Footer etc.
  * 
  */
+
+header{
+    display: flex;
+    justify-content: center;
+    flex-flow:column;
+    align-items: center;
+    height: 150px;
+    background-color: lightseagreen;
+    color: rgb(153, 49, 8);
+    width: 100%;
+}
+nav{
+    background-color: rgb(213, 227, 228);
+    height: 50px;   
+    float:left;
+width: 100%;
+}
+nav li{
+    display: inline-block;
+margin: 5px 10px;
+
+}
+nav a{
+    color: rgb(153, 49, 8);
+
+}
+
+
+footer{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 60px;
+    width: 100%;
+    background-color: lightseagreen;
+}
+main{
+    width: 90%;
+    margin: auto;
+    padding-top: 60px;
+}
+
+article:first-child img{
+  width: 60%;
+  margin-left: 20%;
+}
```

Comment:
Good. In cases where you have given absolute units for values it would probably better to consider using relative units. These take into account the relative size of different elements and the display on which your page is rendered, so it can cope better for example with switching between a small phone display and a desktop computer.

See: https://www.w3schools.com/cssref/css_units.asp

You should also try to keep your code tidy as this makes it easier to read and makes it less likely you'll make other mistakes that stop it working properly. There are instructions on the CYF web site on how to set up automatic formatting in Visual Studio Code which takes care of this:

https://docs.codeyourfuture.io/course-processes/joining-the-course/getting-setup#visual-studio-code

Please let me know if you need any help configuring it.

Reply:
It might be a good idea to limit the width of your `body` element or to add a container element inside `body` that limits the width of the elements on your page as at the moment it can be quite hard to read if the window is maximised.